sechub-pds-solutions/loc/docker/loc-Debian.dockerfile

```diff
@@ -27,9 +30,17 @@ COPY pds-config.json "$PDS_FOLDER"/pds-config.json
 RUN export DEBIAN_FRONTEND=noninteractive && \
     apt-get update && \
     apt-get --assume-yes upgrade && \
-    apt-get --assume-yes install wget perl && \
+    apt-get --assume-yes install perl wget yq && \
     apt-get --assume-yes clean
 
+# Install scc
+RUN cd "$DOWNLOAD_FOLDER" && \
+    wget --no-verbose "https://github.com/boyter/scc/releases/download/v${SCC_VERSION}/scc_${SCC_VERSION}_Linux_x86_64.tar.gz" && \
```

Comment:
Please use the checksums.txt file to cross-check the integrity of the downloaded file: https://github.com/boyter/scc/releases/download/v3.1.0/checksums.txt. Source: https://github.com/boyter/scc/releases/.